lepidopter-fh/etc/ooniprobe.conf

```diff
@@ -62,6 +72,10 @@ tor:
     # This should be set to something to avoid having Tor download each time
     # the descriptors and consensus data.
     data_dir: /opt/ooni/tor_data_dir
+    #
+    # This is the timeout after which we consider to to not have
+    # bootstrapped properly.
+    timeout: 200
 torrc:
     #HTTPProxy: host:port
     #HTTPProxyAuthenticator: user:password
```

Comment:
If you need any extra option be sure to also update the file on lepidopter-update: https://github.com/OpenObservatory/lepidopter-update/blob/master/updater/versions/update-1.py#L40